src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -2530,18 +2572,29 @@ namespace HVACUnitarySystem {
                         UnitarySystem(UnitarySysNum).HeatVolumeFlowRate(Iter) /
                         UnitarySystem(UnitarySysNum).HeatVolumeFlowRate(DesignSpecMSHP(MSHPIndex).NumOfSpeedHeating);
                 }
-                // these coil types do not have an idle speed air flow rate
-                UnitarySystem(UnitarySysNum).IdleVolumeAirRate = 0.0;
-                UnitarySystem(UnitarySysNum).IdleMassFlowRate = 0.0;
-                UnitarySystem(UnitarySysNum).IdleSpeedRatio = 0.0;
```

Comment:
These few lines previously set idle air flow rate to 0 for these coils types. Although these coil types do not have inputs for no load flow rate, the unitary system does, or the DSMSHP object does so this is wrong when constant fan operating mode is used.